Documentation/Signoffs/Power_System_Receiver.md

```diff
@@ -1,37 +1,148 @@
+
+<sup>4</sup> Reducing maintenace is important to reduce the workload for the continuing function of the drone tracker system after the project is completed. For example,the system will be less useful if a worker must constantly change the discharged batteries, so solar generation will be used to recharge batteries that will last for at least two years. <br />
 
-<sup>2</sup> Reason for contraint 2 and why it is important
 
-<sup>3</sup> Reason for contraint 3 and why it is important
 
 ## Schematic
 
+![Screenshot 2024-09-15 at 10 01 20 PM](https://github.com/user-attachments/assets/9f17ad2f-213b-4fd7-9dec-69144ef72f60)<br />
+Figure 1: Electrical Schematic<br />
+The above schematic details the connections and voltages for the system. The solar panel (A) is connected to the Waveshare module (B) Solar In postive and negative terminals at 12 V. This allows the solar panel to recharge the 14500 2500 mAH 3.7 V lithium-ion battery (C) connected to the Waveshare module. The battery is also chargeable using a Micro-USB input, which will be useful for testing purposes. The Waveshare module will power the Arduino Nano ESP32 (D) through the USB out port via a USB-A to USB-C 5 V connection. The module will also power the ESP8266 WiFi module (E) through the 3.3 V output terminals. For this schematic, the full pinout is not detailed for the Arduino Nano ESP32 and the ESP8266 WiFi module. The excluded details are not vital to the functionality of this subsystem; for those details refer to the receiver system documentation or the datasheets provided in the refernces section. <br />
+
+<img width="984" alt="Screenshot 2024-04-10 at 9 43 25 PM" src="https://github.com/mrnye42/Drone-Tracker-Project/assets/158224821/e893c923-8836-458a-af27-fb9d5f76ec00"> <br />
+Figure 2: Waveshare Power Management Module Schematic<br />
+The above figure details the electronics and circuitry found in the Waveshare Power Management Module. The functionality of the Waveshare module is essential to this subsystems design. The various safety features such as over charge and over discharge will also ensure that the design meets safety requirements and improves the longevity of the system and its components. <br />
+
+
 ## Analysis
 
+## Power Requirements
+| Component | Min Voltage (V)| Max Voltage (V)| Min Current (A)| Typical Current (A) | Max Current (A)| Min Power (W)| Typical Power (W)| Max Power (W)|
+| --------- | ---------------| -------------- |----------------| -------------- | ------------ | ------------ | ------------ | ------------ |
+| Arduino Nano ESP32 | 4.8| 5.5 | 0.003 | 0.0783 | 0.25 | 0.0144 | 0.3915 | 1.375 |
+| ESP8266 WiFi Module | 3.3 | 3.3 | 0.015 | 0.07 | 0.215 | 0.0495 | 0.231 | 0.7095 |
+| Total     |                |                |  0.018          |  0.01483         |  0.465          |  0.0639         |  0.6225     |   2.0845      |  <br />
```

Comment:
Be careful with this row - does it make sense to add currents for different source voltages?

Reply:
Removed other source voltages from table.

Reply:
Still not addressed. How is adding currents for different voltage sources relevant? How would one interpret the result - what does it mean?

Reply:
Removed summation of currents for different source voltages.